$OpenBSD: patch-test_run-stp-tests_sh,v 1.5 2009/04/27 08:25:39 bernd Exp $

Second chunk of this patch:
Security fix for CVE-2008-5183.
Patch adapted from Red Hat's solution.

--- test/run-stp-tests.sh.orig	Wed Nov 19 20:50:56 2008
+++ test/run-stp-tests.sh	Sat Apr 18 00:50:57 2009
@@ -21,7 +21,7 @@ argcount=$#
 # Make the IPP test program...
 #
 
-make
+${MAKE_PROGRAM}
 
 #
 # Figure out the proper echo options...
@@ -387,7 +387,7 @@ fi
 
 export LD_LIBRARY_PATH
 
-LD_PRELOAD="$root/cups/libcups.so.2:$root/filter/libcupsimage.so.2"
+LD_PRELOAD="$root/cups/libcups.so.$LIBcups_VERSION:$root/filter/libcupsimage.so.$LIBcupsimage_VERSION"
 export LD_PRELOAD
 
 if test "x$DYLD_LIBRARY_PATH" = x; then
